exercises/raindrops/canonical-data.json

```diff
@@ -1,74 +1,92 @@
 {
    "cases": [
       {
+         "description" : "the sound for 1 is 1",
          "number" : 1,
          "expected": "1"
       },
       {
+         "description" : "the sound for 3 is Pling",
          "number" : 3,
          "expected": "Pling"
       },
       {
+         "description" : "the sound for 5 is Plang",
          "number" : 5,
          "expected": "Plang"
       },
       {
+         "description" : "the sound for 7 is Plong",
          "number" : 7,
          "expected": "Plong"
       },
       {
+         "description" : "the sound for 6 is Pling as it has a factor 3",
          "number" : 6,
          "expected": "Pling"
       },
       {
+        "description" : "the sound for 8 is 8",
```

Comment:
If it's a common mistake in implemention is this exercise, then why not. 

Something like "2 to the power 3 does not make a raindrop sound as 3 is the power not the base"?